docs/reference/mobile/android/setup.md

```diff
@@ -2,36 +2,88 @@
-
-| Field | How to find the path |
-|-------|------|
-| SDK  |  You can find the SDK location by opening Android Studio and selecting **Configure** then **SDK Manager**. |
-| JDK  |  **Windows:** default path is `C:/Program Files/Java` <br/>**Mac:** run `which java` to retrieve the path. |
-| NDK  |  (Optional) Use the directory path that you noted down earlier when extracting the Android NDK.|
+1. Open your SpatialOS project inside the Unity Editor. It is located inside `<path-to-your-project>/workers/unity`.
+
+    > If you don’t have a SpatialOS Unity project, you can use the [FPS Starter Project]({{urlRoot}}/projects/fps/get-started/get-started) or the [Blank Starter Project]({{urlRoot}}/projects/blank/overview) to get started.
+
+1. In your Unity Editor, go to the **External Tools** window. To do this:
+    * on Windows, go to **Edit** > **Preferences** > **External Tools**
+    * on MacOS, go to **Unity** > **Preferences** > **External Tools**
+
+1. In the **Android** section of the **External Tools** window, tick the **Use embedded JDK** checkbox to use the JDK that Unity provides. 
+1. Input the following paths into the **Android Section**:
+
+	| Field | How to find the path |
+	|-------|------|
+	| SDK  |  You can find the SDK location by opening Android Studio and selecting **Configure** then **SDK Manager**. |
+	| NDK  |  (Optional) Use the directory path that you noted down earlier when extracting the Android NDK.|
+
+1. (Optional, if you want to use Unity Remote) Go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window and navigate to the **Unity Remote** section. Set the **Device** setting to **Any Android Device**.
+
+## Set up your project
+
+If you are using one of our [Starter Projects]({{urlRoot}}/reference/glossary#starter-project), you can skip the **Create a mobile connector script** section, as you already have one in your project.
+
+<%(#Expandable title="Create a mobile connector script")%>
+
+If you [added the GDK]({{urlRoot}}/projects/myo/setup) to an existing Unity project rather than using a Starter Project, then you also need to create and add a MonoBehaviour script to your mobile client-worker GameObject. To do this:
+
+1. Create a MonoBehaviour script which inherits from the [`DefaultMobileWorkerConnector`]({{urlRoot}}/api/mobile/mobile-worker-connector) and include the functionality you want. This scripts contains support for both Android and iOS. You can base your implementation on [the one](https://github.com/spatialos/gdk-for-unity-blank-project/blob/develop/workers/unity/Assets/Scripts/Workers/MobileClientWorkerConnector.cs) in our Blank Starter Project.
+1. In your Unity Editor, add the MonoBehaviour script to your mobile client-worker GameObject.
+
+<%(/Expandable)%>
+
+If you want to launch your application on a physical device, you need to 
```

Comment:
I think this belongs in a different section

Reply:
and don't forget the colon